### PR TITLE
SISRP-41163: Converts Housing card to read from new SISEDO view

### PR DESCRIPTION
**Pull request by lyttam**
https://jira.berkeley.edu/browse/SISRP-41163

The Housing card was previously parsing and extracting data from the Financial Aid Data feed. Now it will be populated from a SISEDO view instead.

### CODE REVIEW

**Comment by lyttam** [comment]
@davidlimcheng I removed the testext parameter from those two specs.

File: app/controllers/my_housing_controller.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class MyHousingController < ApplicationController
+  include AllowDelegateViewAs
+  before_filter :api_authenticate_401
+  before_filter :authorize_for_financial
+
+  def get_feed
+    options = params.permit :aid_year
```
Comment (redconfetti) [inline app/controllers/my_housing_controller.rb]: @lyttam - Shouldn't we put some sort of filter in place here that forces this value to be numeric. I see this value ends up inserted into the query directly.

**Comment by redconfetti** [comment]
It's nice to see that we undid that hack I had to apply to get housing data.

File: app/controllers/my_housing_controller.rb (hunk 1 above)
Comment (lyttam) [reply app/controllers/my_housing_controller.rb]: @redconfetti Good idea, I'll add that.

**Comment by lyttam** [comment]
@redconfetti I incorporated your suggestion - see `Finaid::MyHousing#my_aid_year`

**Comment by redconfetti** [comment]
@lyttam - I would have preferred it happen closer to the controller layer, where the content provided by the client/user is received by the system, but this is still good.

I see there is a conflict now with your commits though, so I can't merge this.

**Pull request merged**